hs_tools_resource/static/js/toolresource.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function(){
+    var checked_res_str=$("#checked_res_div").text();
+    var checked_res_array=checked_res_str.split(",");
+    for (var i = 0; i < checked_res_array.length; i++)
+    {
+        var checked_res_item=checked_res_array[i];
+        $("input[value='"+checked_res_item+"']").attr("checked","true");
+    }
+    var get_icon_input_element = function(){
+        return $('#id_icon');
+    };
+    if ($('#icon-preview-img').length == 0) {
+        get_icon_input_element().after('<span id="icon-preview-label" class="control-label">Preview</span><br>' +
+            '<img id="icon-preview-img" src="' + get_icon_input_element().val() + '" style="width:100px; height:100px">');
```

Comment:
If you add this style for image into css as commented above for the template html file, you can probably use addClass() to dynamically add the style class. This way, the style is defined in the single place in css rather than multiple places which could result in out-of-sync issue. Just a suggestion for your consideration.